tools/ARIAtools/ARIAProduct.py

```diff
@@ -322,7 +322,7 @@ def __continuous_time__(self):
                 num_dups.append(i[0])
         # Delete duplicate products
         self.products=list(self.products for self.products,_ in itertools.groupby(self.products))
-        if num_dups!=0:
+        if len(num_dups) > 0:
```

Comment:
@sssangha Can you confirm this change.

Reply:
Idiomatic python would be:
```suggestion
        if num_dups:
```